whimsylib/actor.py

```diff
@@ -170,6 +170,12 @@ def __init__(self, actor_item, *statistics, **kwargs):
     def ai(self):
         return self._ai
 
+    @ai.setter
+    def ai(self, new_ai):
```

Comment:
Should you also call this in https://github.com/Cronenbrogues/whimsylib/pull/11/files#diff-412ce60c55f26cf5fc67b150ecf98ba84b6d4c2dc23398425fada9d5badb7a54R155?